earth2studio/lexicon/hrrr.py

```diff
@@ -37,14 +37,14 @@ class HRRRLexicon(metaclass=LexiconType):
     def build_vocab() -> dict[str, str]:
         """Create HRRR vocab dictionary"""
         sfc_variables = {
-            "u10m": "sfc::anl::10m_above_ground::UGRD",
-            "v10m": "sfc::anl::10m_above_ground::VGRD",
-            "u80m": "sfc::anl::80m_above_ground::UGRD",
-            "v80m": "sfc::anl::80m_above_ground::VGRD",
-            "t2m": "sfc::anl::2m_above_ground::TMP",
-            "refc": "sfc::anl::entire_atmosphere::REFC",
+            "u10m": "sfc::anl::10 m above ground::UGRD",
+            "v10m": "sfc::anl::10 m above ground::VGRD",
+            "u80m": "sfc::anl::80 m above ground::UGRD",
+            "v80m": "sfc::anl::80 m above ground::VGRD",
+            "t2m": "sfc::anl::2 m above ground::TMP",
+            "refc": "sfc::anl::entire atmosphere::REFC",
```

Comment:
@NickGeneva can we add this here? StormCast needs it
```
"mslp": "sfc::anl::mean sea level::MSLMA"
```
(and to the corresponding section in `HRRRFXLexicon`)

Reply:
Added.

FYI, I update the scheme for hybrid levels to {var}{index}hl (e.g. u1hl). No underscores

To match the existing practice for variables like u10m.